frontend/templates/product/sections/ProductOverviewSection/index.tsx

```diff
@@ -219,12 +219,25 @@ export function ProductOverviewSection({
                         product.compatibility.devices.length <= 0
                      }
                   >
-                     <CustomAccordionButton>
-                        Compatibility
-                     </CustomAccordionButton>
-                     <CustomAccordionPanel data-testid="product-compatibility-dropdown">
-                        <CompatibleDevices product={product} />
-                     </CustomAccordionPanel>
+                     {hasCompatibilityNotes ? (
```

Comment:
Any reason not to just use `product.compatibilityNotes` directly?
```suggestion
                     {product.compatibilityNotes ? (
```